### ISSUE TITLE
Relationship to Accessibility specs

### ISSUE THREAD

**Issue by LaurenceZaysser** [mattgarrish/epub-revision#2720]
### Section

https://www.w3.org/TR/epub-33/#sec-intro-relations

### Describe the problem

The section concerning the _Relationship to Other specifications_ is absolutely great.
But a section on the _Relationship to Accessibility specs_ (WCAG/ARIA/EPUB -A11Y) is missing there, though there is a section 10 Accessibility further down.

- Report to WCAG https://www.w3.org/TR/WCAG22/
- Report to ARIA roles https://www.w3.org/TR/wai-aria-1.2/#role_definitions
- Report to DPUB-ARIA roles https://www.w3.org/TR/dpub-aria-1.1
- Report to https://www.w3.org/TR/epub-33/#sec-accessibility (or move this section 10 into section 1.3 ?)

**Comment by mattgarrish** [comment]
The relationships section is meant to explain specific implementation details or differences with the underlying technologies used in EPUB. In the case of WCAG, ARIA, etc. those are technologies that HTML and SVG allow but there aren't implementation details of them that need to be called out by the EPUB standard.

The accessibility specification has a relationship to WCAG section, for example, because that specification builds on WCAG and the section explains how the two relate. ARIA and DPUB-ARIA are technologies for implementing accessibility, but they're not being built on by either standard.

That said, we probably could look at making more mention of DPUB-ARIA, maybe in the main accessibility section. It's kind of a lost reference in the epub:type caution note.

**Comment by LaurenceZaysser** [comment]
OK for adding at least a TOC entry named "_Relationship to ARIA-DPUB_" as you suggest.
Though I am afraid it too restrictive. The relationship to ARIA (roles & states) must be clarified, as well as the relationship to WCAG by the way.
Some people still maintain that EPUB accessibility is only defined by EPUB A11Y spec, and consider that neither WCAG nor ARIA apply to EPUBs!... :-(

**Comment by mattgarrish** [comment]
> OK for adding at least a TOC entry named "_Relationship to ARIA-DPUB_" as you suggest.

No, I'm only suggesting that we can mention DPUB-ARIA in the existing accessibility section. DPUB-ARIA isn't a foundational technology that the EPUB specification builds on.

**Comment by LaurenceZaysser** [comment]
People do not care about the past. They do care about the present, and the patchwork of technologies EPUB reuses (without redefining it), in order to get the full picture.
DPUB-ARIA (together with ARIA) is much important for Assistive Technologies that do not use epub:type attribute, which is dedicated to reading systems.

**Comment by iherman** [comment]
This was discussed during the [pmwg meeting on 10 July 2025](https://w3c.github.io/pm-wg/minutes/2025-07-10.html#9f66).

<details><summary><i>View the transcript</i></summary>

<h3>Relationship to Accessibility Specs - <a href="https://github.com/w3c/epub-specs/issues/2720" rel="noopener noreferrer">w3c/<wbr>epub-specs#2720</a></h3>
<p><cite>wendy:</cite> the desire is to make clear relationship with WCAG/ARIA/EPUB-A11Y.</p>
<p><cite>DaleRogers:</cite> what is our lane, how do we stay in our lane? when we write the spec, we have different audiences, how do we care of them?</p>
<p><cite>ivan:</cite> the spec is a tech document, the target is tech people producing ebooks, tools and reading systems. We probably should have a different document on how to create epubs.<br>
<span>… what Matt is seing is that the section refers to stds that are directly relevant for EPUB, not those which are indirectly referenced, like SVG (which depends on HTML). EPUB depends on the sister stds, like the accessibility docs.</span><br>
<span>… listing all others is going to far.</span></p>
<p><cite>AvneeshSingh:</cite> epub a11y, wcag put constraints on EPUB. HTML SVG are building blocks. DPUB ARIA is a dependency, the other are not adding to the standard.</p>
<p><cite>CharlesL:</cite> I see the point. Something is missing. To make an accessible EPUB you need these things. EPUB has a relationship with DPUB ARIA. I would be in favor of adding it to the A11Y spec.</p>
<p><cite>wendyreid:</cite> I think we didn't do that before because it was a draft. We can reference it now. There is a one-sentence in the spec; maybe we should add to it.<br>
<span>… our audience is mixed. We made the spec readable. We should listen to feedback.</span></p>
<p><cite>ivan:</cite> EPUB could reference the a11y spec and the reading system spec; and the a11y spec can reference DPUB ARIA. For me it is a different level.</p>
<p><cite>AvneeshSingh:</cite> DPUB ARIA is designed to provide semantic info to EPUB, it is a building block.<br>
<span>… ARIA is going beyond a11y.</span></p>
<p><cite>ivan:</cite> why no rdf-a in this case? it is used in html.</p>
<p><cite>wendyreid:</cite> RS process DPUB ARIA. It is impactful for content creators, more than rdf-a.<br>
<span>… next topic</span></p>

<hr /></details>

**Comment by mattgarrish** [comment]
Maybe the minutes aren't clear, but I think there's still a misunderstanding of what the relationship sections are for. It's not a place to list things that you can use but aren't fundamental to the standard. DPUB-ARIA does not place any restrictions on EPUB. It's not even a requirement for meeting the accessibility specification.

DPUB-ARIA has a relationship to ARIA because it is a module that extends it (but we didn't bother making a relationship section since that's so fundamental it's just part of the introduction to that specification). That's the kind of relationship that this sort of section is meant to note. It's not the place to list any other technology you could possibly use. Otherwise, we need to start running down every possible technology you can use in EPUB, from rdfa, to jpeg, to png, to mp3, to web assembly, and on and on.

**Comment by mattgarrish** [comment]
And to be clear, I still think we can improve the accessibility section to mention DPUB-ARIA, and probably look at the accessibility standard, too, but the "relationship to" section is not the place to add this kind of info.
